### PR TITLE
Feature 1566 discountrule hasorderedxtimes

### PR DESCRIPTION
**Pull request by orrTechnologies**
Plugin will allow you to use promotions based on how many times a customer has ordered IE Free shipping for first order only.

### CODE REVIEW

File: src/NopCommerce.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
```
Comment (odises) [inline src/NopCommerce.sln]: Why did you change the sln file?!!

Comment (orrTechnologies) [inline src/NopCommerce.sln]: Ugh must of missed it in my commit

**Comment by AndreiMaz** [comment]
Thanks a lot. But we do not add new plugins to the core. Could you please create a new plguin and upload it to http://www.nopcommerce.com/extensions-and-themes.aspx? as described at http://www.nopcommerce.com/contribute.aspx
